contracts/Unibright.sol

```diff
@@ -255,10 +268,13 @@ contract UBTSplitter is Context, Ownable {
 
         validatorStakingAddress[revenueAddress] = stakingAddress;
```

Comment:
do we need some check if payee already exists before updating it? i tried to call this function in test with accounts[5] which is not used, and it worked, even though accounts[5] payee was not added before 

if we do this, reminder to not forget to add tests

Reply:
Also agree, let's extract the logic for updating shares, event nonce and periods to a private function to avoid duplication